go.mod

```diff
@@ -5,11 +5,11 @@ go 1.20
 require (
 	github.com/gorilla/mux v1.8.0
 	github.com/joho/godotenv v1.5.1
-	go.mongodb.org/mongo-driver v1.11.6
-	golang.org/x/text v0.3.8
```

Comment:
I think you accidentally reverted Josh's change here to which was to update the text library which was giving us an error

Reply:
might be stuff in the go.sum too